decode.rb

```diff
@@ -0,0 +1,38 @@
+@decoder_letters = { '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E', '..-.' => 'F', '--.' => 'G',
+                     '....' => 'H', '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N',
+                     '---' => 'O', '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T', '..-' => 'U',
+                     '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z', '.----' => '1',
+                     '..---' => '2', '...--' => '3', '....-' => '4', '.....' => '5', '-....' => '6', '--...' => '7',
+                     '---..' => '8', '----.' => '9', '-----' => '0' }
+
+# method to decode a char
+def decode_char(string)
+  @decoder_letters.fetch(string)
+end
+
+puts decode_char('.-')
```

Comment:
- [ ] Based on the requirements of this project, this method should return **"A"** with quotes and not **A** without quotes as in your case.